onnxscript/autocast.py

```diff
@@ -83,13 +83,11 @@ def cast(x, typeinfo) -> tensor.Tensor:
             return tensor.Tensor(np.array(x, dtype=dtype))
         return x
 
-    return cast_inputs(get_type_info, cast, op_schema, *args)
+    return cast_inputs(get_type_info, cast, op_schema, args)
 
 
-def static_cast_inputs(converter, op_schema: Optional[OpSchema], *args):
+def static_cast_inputs(converter, op_schema: Optional[OpSchema], *args) -> tuple[str, ...]:
```

Comment:
Should this take `args` instead of `*args` too?

Reply:
Done